src/main/java/net/conjur/api/clients/AuthnK8sClient.java

```diff
@@ -0,0 +1,21 @@
+package net.conjur.api.clients;
+import net.conjur.api.AuthnProvider;
+import net.conjur.api.Token;
+
+public class AuthnK8sClient implements AuthnProvider {
```

Comment:
I agree with @RafiS3, that we should get this to line up with our Go API client. I think we can do this iteratively though, lining it up with the `TokenAuthenticator` [1], rather than the `TokenFileAuthenticator`. I would suggest we rename this to `AuthnTokenProvider`. We can then add a ticket to add a `AuthnTokenProvider` that includes the file watch support.

[1] https://github.com/cyberark/conjur-api-go/blob/b75a31bafc95e06780b81df18ba5dba8dd0dc32f/conjurapi/authn/token_authenticator.go

Reply:
I will add that I think the should be updated to include the environment variable, but I would suggest we use `CONJUR_AUTHN_TOKEN` rather than the file path for this PR.